src/main/java/by/itechart/film2night/dao/impl/CountryDaoImpl.java

```diff
@@ -0,0 +1,66 @@
+package by.itechart.film2night.dao.impl;
+
+import by.itechart.film2night.connection.ConnectionPool;
+import by.itechart.film2night.dao.CountryDao;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class CountryDaoImpl implements CountryDao {
+
+    private static final String FIND_ID_BY_NAME = "SELECT id_country FROM kinopoiskdb.country  WHERE name_country=?";
+    private static final Integer DefaultCountryId = 1;
```

Comment:
variableName should be in camelCase or VARIABLE_NAME in upper case for constants